modules/nw-dns-forward.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * dns/dns-operator.adoc
+
+[id="nw-dns-forward_{context}"]
+= Using DNS forwarding
+
+You can use DNS forwarding to override the forwarding configuration identified in `etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone.
+
+.Procedure
+
+. Modify the DNS Operator object named `default`:
++
+----
+$ oc edit dns.operator/default
+----
+.. This allows the Operator to create and update the ConfigMap named `dns-default` with additional server configuration blocks based on `Server`. If none of the servers has a zone that matches the query, then name resolution falls back to the name servers that are specified in `/etc/resolv.conf`.
++
+.Sample DNS
+[source,YAML]
+----
+apiVersion: operator.openshift.io/v1
+kind: DNS
+metadata:
+  name: default
+spec:
+  servers:
+  - name: foo-server <1>
+    zones: <2>
+      - foo.com
+    forwardPlugin:
+      upstreams: <3>
+        - 1.1.1.1
+        - 2.2.2.2:5353
+  - name: bar-server
+    zones:
+      - bar.com
+      - example.com
+    forwardPlugin:
+      upstreams:
+        - 3.3.3.3
+        - 4.4.4.4:5454
+----
+<1> `Name` must comply with the `rfc6335` service name syntax.
+<2> `Zones` must conform to the definition of a `subdomain` in `rfc1123`. The cluster domain, `cluster.local`, is an invalid `subdomain` for `Zones`.
+<3> A maximum of 15 `Upstreams` is allowed per `ForwardPlugin`.
++
+[NOTE]
+====
+If `Server` is undefined or invalid, the ConfigMap only contains the default server.
```

Comment:
Just wondering what this `Server` item references. I don't see anything in the example yaml that has a capital `Server`.

Same question in the `Server` reference in the step above.